types/node/index.d.ts

```diff
@@ -1151,7 +1151,7 @@ declare module "http" {
         constructor();
 
         setTimeout(msecs: number, callback?: () => void): this;
-        destroy(error: Error): void;
+        destroy(error?: Error): void;
```

Comment:
Is `destroy()`  really needed here? Base class `stream.Writeable` has `destroy()` already.